data/tutorials/getting-started/3_01_ocaml_on_windows.md

```diff
@@ -6,8 +6,211 @@ description: >
 category: "Resources"
 ---
 
-There is a [DkML] Windows
-installer that we recommend for new users. However, while [DkML] has a modern OCaml 4.14.0 compiler,
+## opam
+
+Opam now features a fully native Windows compatible installation process that we
+recommend new users to follow below.
+
+In order to have access to OCaml on a Windows PC, users have to use opam so that's our
+first step.
+
+Windows Developers out there will already be familiar with `winget` which is
+the default package installer for Windows. Opam is now distributed with it. 
+
+```shell-session
```

Comment:
Thanks for showing the output! Makes it easy for user to follow and also shows them it is not complicated.